docker-compose.yml

```diff
@@ -95,6 +95,21 @@ services:
       - WONDERWALL_UPSTREAM_HOST=skribenten-bff:8084
       - WONDERWALL_LOG_LEVEL=debug
       - WONDERWALL_LOG_FORMAT=text
+  texas:
+    profiles:
+      - skribenten
+    image: ghcr.io/nais/texas:latest
+    ports:
+      - "3000:3000"
```

Comment:
Texas trenger vel ikke å eksponeres utenfor for docker-nettverket? Det er vel kun bff som trenger å snakke med den?